grpc-spring-boot-starter/build.gradle

```diff
@@ -286,6 +286,8 @@ dependencies {
     compileOnly "org.springframework.boot:spring-boot-starter-actuator"
     compileOnly "org.springframework.boot:spring-boot-starter-validation"
     compileOnly 'org.springframework.cloud:spring-cloud-starter-consul-discovery'
+    compileOnly 'io.opentelemetry:opentelemetry-exporter-otlp:1.29.0'
+    compileOnly 'io.micrometer:micrometer-tracing-bridge-otel:1.1.4'
```

Comment:
`io.micrometer:micrometer-tracing-bridge-otel` is it in-use ?

Reply:
<img width="819" alt="image" src="https://github.com/LogNet/grpc-spring-boot-starter/assets/31073930/b0b170e4-a49a-41aa-ae59-4eb231752609">
Used in OtelConfiguration, removed from client